webviz_ert/views/parameter_view.py

```diff
@@ -9,7 +9,7 @@
 import webviz_ert.assets as assets
 
 
-def parameter_view(parent: WebvizPluginABC, index: int = 0) -> List[Component]:
+def parameter_view(parent: WebvizPluginABC, index: str = "") -> List[Component]:
```

Comment:
Why did you change this index from int to str ?

Reply:
because it seems like we are using a string when we call this function, and `mypy` complaints if we leave it as `int` in the function signature :thinking: 
i tried to figure out how and what, but then I just quickly tested if this works, and it makes mypy happy, and nobody else complained either, and the site still worked fine. but i did not try to figure out if this actually needs to be an int...

Reply:
i'm gonna go ahead and merge, as no exceptions were thrown, all tests passed, and no functionality seemed removed based on manual visual testing due to this particular change.